pkg/go/Makefile

```
@@ -48,4 +44,4 @@ format: $(GO_BIN)/gofumpt ## Format Go source files
 	@echo "==> Formatting project files"
 	gofumpt -w transformer/ errors/
 
-all-tests: build audit lint test ## Run linting and tests
\ No newline at end of file
+all-tests: build audit lint test ## Run linting and tests
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)